srcpkgs/gts/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'gts'
+pkgname=gts
+version=0.7.6
+revision=1
+build_style=gnu-configure
+hostmakedepends="glib-devel pkg-config"
+depends="glib"
```

Comment:
Should be detected by default via shlibs

Reply:
Oh cool!